tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier/Program.cs

```diff
@@ -7,33 +7,24 @@ namespace Microsoft.AspNet.Server.Kestrel.LibuvCopier
 {
     public class Program
     {
-        public void Main(string[] args)
+        public static void Main(string[] args)
         {
             try
             {
-                var packagesFolder = Environment.GetEnvironmentVariable("DNX_PACKAGES");
+                var packagesFolder = Environment.GetEnvironmentVariable("DOTNET_PACKAGES");
 
                 if (string.IsNullOrEmpty(packagesFolder))
                 {
-                    var dnxFolder = Environment.GetEnvironmentVariable("DNX_HOME") ??
-                                    Environment.GetEnvironmentVariable("DNX_USER_HOME") ??
-                                    Environment.GetEnvironmentVariable("DNX_GLOBAL_HOME");
-
-                    var firstCandidate = dnxFolder?.Split(';')
-                                                  ?.Select(path => Environment.ExpandEnvironmentVariables(path))
-                                                  ?.Where(path => Directory.Exists(path))
-                                                  ?.FirstOrDefault();
-
-                    if (string.IsNullOrEmpty(firstCandidate))
-                    {
-                        dnxFolder = Path.Combine(GetHome(), ".dnx");
-                    }
-                    else
-                    {
-                        dnxFolder = firstCandidate;
-                    }
+                    packagesFolder = Environment.GetEnvironmentVariable("NUGET_PACKAGES");
```

Comment:
all this dance was required to make sure we can find packages on dev boxes and different CIs (TC/travis etc.) - make sure that CIs pass before pushing this...